deployment/CP14SEPM_Deploy.csv

```diff
@@ -1,12 +1,12 @@
-#AR82,,,,CP14SEPM-SBS01-01-MOPAK0000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,,,35.8514,-74.8482,,0,300,,,
-AR82,,,,CP14SEPM-SBI01-02-CTDMOS011,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,,CGINS-CTDMOS-13559,35.8514,-74.8482,,1,300,,,
-AR82,,,,CP14SEPM-RII01-02-ADCPUP021,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,,CGINS-ADCPUP-00152,35.8514,-74.8482,,17,300,,,
-AR82,,,,CP14SEPM-RII01-02-ADCPSL010,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,,CGINS-ADCPSL-21497,35.8514,-74.8482,,281,300,,,
-AR82,,,,CP14SEPM-WFP01-00-WFPENG001,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGCON-CWFPCP-30002,35.8514,-74.8482,,280,300,,,
-AR82,,,,CP14SEPM-WFP01-01-VEL3DK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-VEL3DK-00470,35.8514,-74.8482,,280,300,,,
-AR82,,,,CP14SEPM-WFP01-02-DOFSTK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-DOFSTK-03523,35.8514,-74.8482,,280,300,,,
-AR82,,,,CP14SEPM-WFP01-03-CTDPFK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,35.8514,-74.8482,,280,300,,,
-AR82,,,,CP14SEPM-WFP01-04-FLORTK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,35.8514,-74.8482,,280,300,,,
-AR82,,,,CP14SEPM-WFP01-05-PARADK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-PARADK-20609,35.8514,-74.8482,,280,300,,,
+AR82,,,,CP14SEPM-SBS01-00-STCENG000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,,CGCON-BSTC01-00016,35.8514,-74.8482,,0,300,,,A
+AR82,,,,CP14SEPM-SBS01-05-MOPAK0000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,,CGINS-MOPAK0-99999,35.8514,-74.8482,,0,300,,,A
+AR82,,,,CP14SEPM-SBI01-02-CTDMOS011,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,,CGINS-CTDMOS-13559,35.8514,-74.8482,,1,300,,,A
+AR82,,,,CP14SEPM-RII01-02-ADCPUP021,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,,CGINS-ADCPUP-00152,35.8514,-74.8482,,17,300,,,A
+AR82,,,,CP14SEPM-RII01-02-ADCPSL010,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,,CGINS-ADCPSL-21497,35.8514,-74.8482,,281,300,,,A
+AR82,,,,CP14SEPM-WFP01-00-WFPENG001,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGCON-CWFPCP-30002,35.8514,-74.8482,,280,300,,,A
+AR82,,,,CP14SEPM-WFP01-01-VEL3DK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-VEL3DK-00470,35.8514,-74.8482,,280,300,,,A
+AR82,,,,CP14SEPM-WFP01-02-DOFSTK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-DOFSTK-03523,35.8514,-74.8482,,280,300,,,A
+AR82,,,,CP14SEPM-WFP01-03-CTDPFK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,35.8514,-74.8482,,280,300,,,A
+AR82,,,,CP14SEPM-WFP01-04-FLORTK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,35.8514,-74.8482,,280,300,,,A
+AR82,,,,CP14SEPM-WFP01-05-PARADK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-PARADK-20609,35.8514,-74.8482,,280,300,,,A
```

Comment:
Does SEPM need Rev B instead of Rev A? If 99999 is the stand-in for the unknown MOPAK s/n, everything else looks good.

Reply:
Rebecca answered my questions on Slack. Looks good!